First commits (#2)
cisagov/guacscanner-docker: README.md

1. Pull the new image:

    ```console
    docker pull cisagov/guacscanner:0.0.1-rc.18
    ```

1. Recreate and run the container by following the [previous
   instructions](#running-with-docker).

## Image tags ##

The images of this container are tagged with [semantic
versions](https://semver.org) of the underlying example project that
they containerize.  It is recommended that most users use a version
tag (e.g. `:0.0.1-rc.18`).

| Image:tag | Description |
|-----------|-------------|

> |`cisagov/guacscanner:1.2.3`| An exact release version. |
> |`cisagov/guacscanner:1.2`| The most recent release matching the major and minor version numbers. |
> |`cisagov/guacscanner:1`| The most recent release matching the major version number. |

Reviewer: Match current version:
```suggestion
|`cisagov/guacscanner:0.0.1`| An exact release version. |
|`cisagov/guacscanner:0.0`| The most recent release matching the major and minor version numbers. |
|`cisagov/guacscanner:0`| The most recent release matching the major version number. |
```

Author: I know we do this elsewhere, but looking at it this time I think the default values (because the major, minor, and patch values are all different from one another) make the meaning behind each of the the tags clearer.  If the version is `0.0.1`, and you check out `cisagov/guacscanner:0`, it isn't clear a priori that the zero in the image tag refers to the major version.

This is a subtle point, but I think it is an important one.

Reviewer: I see what you're saying, but I still think that this is clear when coupled with the corresponding explanatory description text.  I like always showing in the docs what the current versions are, rather than tags that may or may not exist or be the latest versions.  I think we need @mcdonnnj and @felddy to weigh in here.

Author: I'm not convinced it's best, but I will defer to the inertia of the way we have been doing it.   Please enjoy commit 2446f05.

Reviewer: Maybe open an issue or discussion for this in skeleton-docker?

|`cisagov/guacscanner:edge` | The most recent image built from a merge into the `develop` branch of this repository. |
|`cisagov/guacscanner:nightly` | A nightly build of the `develop` branch of this repository. |
|`cisagov/guacscanner:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |

See the [tags tab](https://hub.docker.com/r/cisagov/guacscanner/tags)
on Docker Hub for a list of all the supported tags.

## Volumes ##

There are no volumes.

<!-- | Mount point | Purpose        | -->
<!-- |-------------|----------------| -->

> <!-- | `/var/log`  |  Log storage   | -->

Reviewer 2: I think it's better to use a single logical comment block. This also mirrors how the required environment variable section is commented out.

```suggestion
<!--
| Mount point | Purpose        |
|-------------|----------------|
| `/var/log`  |  Log storage   |
-->
```

## Ports ##

No ports are exposed by this container.

> <!-- | Port | Purpose        | -->
> <!-- |------|----------------| -->
> <!-- | 8080 | Example only; nothing is actually listening on the port | -->

Reviewer 2: ```suggestion
<!--
| Port | Purpose        |
|------|----------------|
| 8080 | Example only; nothing is actually listening on the port |
-->
```

> <!-- The sample [Docker composition](docker-compose.yml) publishes the -->
> <!-- exposed port at 8080. -->

Reviewer 2: ```suggestion
<!--
The sample [Docker composition](docker-compose.yml) publishes the
exposed port at 8080.
-->
```

### Optional ###

There are no required environment variables.

> <!-- | Name  | Purpose | Default | -->
> <!-- |-------|---------|---------| -->
> <!-- | `ECHO_MESSAGE` | Sets the message echoed by this container.  | `Hello World from Dockerfile` | -->

Reviewer 2: ```suggestion
<!--
| Name  | Purpose | Default |
|-------|---------|---------|
| `ECHO_MESSAGE` | Sets the message echoed by this container.  | `Hello World from Dockerfile` |
-->
```

## Secrets ##

| Filename     | Purpose |
|--------------|---------|
| postgres-username | Text file containing the username of the `postgres` user used by the `guacamole` container |
| postgres-password | Text file containing the password of the `postgres` user used by the `guacamole` container |

> | private_ssh_key | Text file containing the private ssh key to use for SFTP file transfer in Guacamole. |

Reviewer: Capitalization:
```suggestion
| private_ssh_key | Text file containing the private SSH key to use for SFTP file transfer in Guacamole. |
```

### Optional ###

> There are no required environment variables.

Reviewer 2: ```suggestion
There are no optional environment variables.
```

<!--
| Name  | Purpose | Default |
|-------|---------|---------|
| `ECHO_MESSAGE` | Sets the message echoed by this container.  | `Hello World from Dockerfile` |
-->

## Secrets ##

| Filename     | Purpose |
|--------------|---------|

> | postgres-username | Text file containing the username of the `postgres` user used by the `guacamole` container |
> | postgres-password | Text file containing the password of the `postgres` user used by the `guacamole` container |

Reviewer 2: Just for consistency.

```suggestion
| postgres-username | Text file containing the username of the `postgres` user used by the `guacamole` container. |
| postgres-password | Text file containing the password of the `postgres` user used by the `guacamole` container. |
```
